### PR TITLE
Remove unused CM headers

### PR DESCRIPTION
**Pull request by onabiull**

### CODE REVIEW

File: contrib/cm/include/cm_common.h [hunk 1]
```diff
@@ -1,106 +1,88 @@
...
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+* THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+* OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+* ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+* OTHER DEALINGS IN THE SOFTWARE.
+*/
 
-#ifndef CM_COMMON_H
-#define CM_COMMON_H
+#ifndef CMRTLIB_LINUX_SHARE_CM_COMMON_H_
```
Comment (dvrogozh) [inline contrib/cm/include/cm_common.h]: Why? Usually protection defines are named in accordance to the file name. Or simply use `#pragma once`.

File: contrib/cm/include/cm_def.h [hunk 2]
```diff
@@ -1,1715 +1,770 @@
...
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+* OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+* THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+* OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+* ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+* OTHER DEALINGS IN THE SOFTWARE.
+*/
+#ifndef CMRTLIB_AGNOSTIC_SHARE_CM_DEF_H_
```
Comment (dvrogozh) [inline contrib/cm/include/cm_def.h]: same?

File: contrib/cm/include/cm_def.h [hunk 3]
```diff
@@ -1,1715 +1,770 @@
...
+#define CM_DDI_2_4 204
+#define CM_DDI_3_0 300
+#define CM_DDI_4_0 400
+#define CM_DDI_5_0 500
+#define CM_DDI_6_0 600
+#define CM_DDI_7_0 700
+#define CM_DDI_7_2 702 //for MDFRT API refreshment.
+
+//Error code definition
+typedef enum _CM_RETURN_CODE
```
Comment (dvrogozh) [inline contrib/cm/include/cm_def.h]: what's wrong with diff? why so many differences? (any line feed changes?)

**Comment by onabiull** [comment]
All three files are exact copies from media-driver.

**Pull request merged**